docs/get-started/full-node-deployment.md

```diff
@@ -190,24 +190,40 @@ KillSignal=SIGINT
 TimeoutStopSec=120
 LimitNOFILE=65535
 LimitNPROC=65535
+PrivateTmp=true
+# MemoryMax setting replaces MemoryLimit in newer systemd versions
```

Comment:
`Suggested flags` ?

Reply:
[PrivateTmp](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#PrivateTmp=) 
create private `/tmp`, `/var/tmp` directory of process, strengthen security